OpenTelemetry.Dotnet.Auto.psm1

```diff
@@ -287,9 +324,25 @@ function Unregister-OpenTelemetryForWindowsService() {
     Remove-Windows-Service -WindowsServiceName $WindowsServiceName
 }
 
+<#
+    .SYNOPSIS
+    Locates OpenTelemetry .NET Automatic Instrumentation's install path. 
+#>
+function Get-OpenTelemetryInstallDirectory() {
+    $installDir = Get-Current-InstallDir
+
+    if ($installDir) {
+        return $installDir
+    }
+
+    Write-Warning "OpenTelemetry .NET Automatic Instrumentation is not installed."
+}
+
 Export-ModuleMember -Function Install-OpenTelemetryCore
 Export-ModuleMember -Function Register-OpenTelemetryForIIS
 Export-ModuleMember -Function Register-OpenTelemetryForWindowsService
+Export-ModuleMember -Function Register-OpenTelemetryForCurrentSession
```

Comment:
Can you also add `Unregister-OpenTelemetryForCurrentSession`?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1473/commits/fe06b8f6dfbd72d3186e5bb138e7e7a8fbe44bce